### DISCUSSION TITLE
Want to hire someone to set up a site

### DISCUSSION THREAD

**Discussion by voyager2222** [community/community#145737] [New to GitHub]
I'm assured I can follow the directions on my own but I'd rather have someone who already knows how to do it. Get in touch in you're interested.

**Comment by goldentalent626** [comment]
I am interested

**Comment by voyager2222** [comment]
Howdy -

I'm a writer who wrote a lot of comics. Twenty years ago I built a site with only HTML listing my credits, with links to images of covers. That's it. It serves primarily as a reference for my readers, and almost never changes. I was on GoDaddy until they switched from Windows to Linux and helpfully put all my files in a folder I can't access. When I complained, two different agents told me they had no responsibility for fixing that. I went to Bluehost and discovered that most hosts now want Wordpress-built sites; they couldn't even handle straight HTML. So now I need a new home. My knowledge of web building began and ended 20 years ago, and that’s why I’d like someone currently knowledgable to set this one up.

What would you charge, and what else might I need to know?

Thanks for responding.

**Comment by goldentalent626** [comment]
Hi.
I think it is very simple for me and i can do it asap.
This will be our first contract, so price is depends on you.
I can start work right now.
If it is possible, you can create a new job on upwork or other website.
Thanks

**Comment by voyager2222** [comment]
Sounds good - but I don’t know what you mean “create a new job…” And I’m happy to pay anything reasonable but I don’t know what “reasonable” is. Just assume I’m a complete idiot.

**Reply by goldentalent626** [reply]
How about chat on other platform?
If you suggest, i will send msg.

**Comment by goldentalent626** [comment]
I mean, if you post a job on upwork or other site, I can work via that platform.
If you cant, I can work without that platform.
Is it okay?

**Comment by voyager2222** [comment]
Okay, RW  -
 
I’m about to run out for a few hours of errands (and tomorrow is Thanksgiving in the US, if you’re not there). So I won’t be able to keep up a continuous conversation. But you can use my email ***@***.*** ***@***.***>) to msg me.

Again, I don’t know how any of this works. I did know how it worked in 2003, but now I just want a site that's set up so I can upload my html files and jpgs and be good to go.

**Reply by goldentalent626** [reply]
I cant see your gmail, i am not sure why that was hidden like this.
Can you send me with screenshot?
![image](https://github.com/user-attachments/assets/265d9075-fb61-4598-8187-03dbf6b5eb6a)

**Comment by voyager2222** [comment]
Does this work?

It’s

e
h
a
r
t
s
@
p
a
c
b
e
l
l
.
n
e
t

**Comment by goldentalent626** [comment]
Is this right?
![image](https://github.com/user-attachments/assets/d826d757-afcb-4c96-a448-5bf7e7dc2c76)

**Comment by voyager2222** [comment]
YepSent from my iPhoneOn Nov 27, 2024, at 12:59 PM, rare-wolf ***@***.***> wrote:﻿
Is this right?
image.png (view on web)

**Comment by ghostinhershell** [comment]
Hi @voyager2222, we're happy you're here 😀

However, we currently do not allow job listings on the Community Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
